WEBHOOKS.md

```diff
@@ -218,3 +226,64 @@ function when comparing signatures.
   createdAt: string;
 }
 ```
+
+- <a id="comment-created-event">**COMMENT_CREATED**</a>
+
+```ts
+{
+  id: string;
+  type: "COMMENT_CREATED";
+  tenantID: string;
+  tenantDomain: string;
+  data: {
+    /**
+     * storyID is the ID of the newly created Story.
+     */
+    storyID: string;
+
+    /**
+     * storyURL is the URL of the newly created Story.
+     */
+    storyURL: string;
+
+    /**
+     * siteID is the Site that the newly created Story was created on.
+     */
+    siteID: string;
+  }
+  createdAt: string;
+}
+```
+
+- <a id="comment-reply-created-event">**COMMENT_REPLY_CREATED**</a>
+
+```ts
+{
+  id: string;
+  type: "COMMENT_REPLY_CREATED";
+  tenantID: string;
+  tenantDomain: string;
+  data: {
+    /**
+     * storyID is the ID of the newly created Story.
+     */
+    storyID: string;
+
+    /**
+     * storyURL is the URL of the newly created Story.
+     */
+    storyURL: string;
+
+    /**
+     * siteID is the Site that the newly created Story was created on.
+     */
+    siteID: string;
+
+    /**
+     * commentID is the comment to which the new comment was a reply
+     */
+    commentID: string;
+  }
+  createdAt: string;
```

Comment:
As already mentioned, the comments should be adapted and we should probably double check the data that is being sent out.